packages/salesforcedx-slds-linter/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@salesforce/salesforcedx-slds-linter",
   "displayName": "SFDX SLDS Linting for VS Code",
   "description": "Provides linting for deprecated SLDS class names",
-  "version": "40.9.0",
+  "version": "40.10.0",
```

Comment:
@chowwinston - How come this was not fixed in develop after your merge yesterday? If this change was not in, it would have caused build issues, I think.